src/compiler/build/build-hmr.ts

```diff
@@ -205,7 +205,17 @@ const getImagesUpdated = (buildCtx: d.BuildCtx, outputTargetsWww: d.OutputTarget
   return imageFiles.sort();
 };
 
-const excludeHmrFiles = (config: d.Config, excludeHmr: string[], filesChanged: string[]) => {
+/**
+ * Determine a list of files (if any) which should be excluded from HMR updates. 
+ *
+ * @param config a user-supplied config
+ * @param excludeHmr a list of glob patterns that should be used to determine
+ * whether to exclude a file or not (a file will be excluded if it matches one
```

Comment:
Minor, missing parens
```suggestion
 * whether to exclude a file or not (a file will be excluded if it matches one)
```